beetmoverscript/data/beetmover_task_schema.json

```diff
@@ -49,6 +49,9 @@
                         "hashType":  {
                             "type" : "string"
                         },
+                        "stage_platform":  {
+                            "type" : "string"
+                        },
```

Comment:
Hm, I don't get this, why is this needed? And if so, is it required or optional?

Reply:
By my understanding, stage_platform is a hack that maps stage platform names and is needed in releaseProperties for the time being https://github.com/lisajguo/beetmoverscript/blob/master/beetmoverscript/constants.py#L15

It's optional because: https://github.com/lisajguo/beetmoverscript/blob/master/beetmoverscript/task.py#L165

Reply:
Will update the schema to explicitly state that it's optional